ui/src/index.html

```diff
@@ -21,6 +21,17 @@
         google.charts.load('current', {'packages': ['corechart']});
     </script>
 
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-112406425-1"></script>
+   <script>
```

Comment:
NIT: this tag and the closing `<script>` tag need to be indented another space plz.

Reply:
Done.